pkgs/version/version.go

```diff
@@ -30,14 +29,18 @@ func GetLatestTag(environment string) (string, error){
 }
 
 func TrimTag(latestTagRaw string) string {
-	log.Println("Trimming tag")
+	log.Println("Trimming tag:", latestTagRaw)
 	latest_tag := strings.Trim(latestTagRaw, "\n")
 	latest := strings.Trim(latest_tag, " ")
+	// After trimming - check if no previous tag exists, and return fallout tag
```

Comment:
please add a log here tag tell that latest was not found and we used the default tag
something like that:
log.Println("Lastest tag was not found using default")